frontend/src/concepts/pipelines/content/configurePipelinesServer/utils.ts

```diff
@@ -100,11 +100,16 @@ export const createDSPipelineResourceSpec = (
       awsRecord.AWS_S3_ENDPOINT?.match(/^(?:(\w+):\/\/)?(.*)/) ?? [];
 
     return {
+      dspVersion: 'v2',
+      mlpipelineUI: {
+        image: 'quay.io/opendatahub/ds-pipelines-frontend:latest', // TODO: remove this before release
+      },
       objectStorage: {
         externalStorage: {
           host: externalStorageHost?.replace(/\/$/, '') || '',
           scheme: externalStorageScheme || 'https',
           bucket: awsRecord.AWS_S3_BUCKET || '',
+          region: 'us-east-2', // TODO hardcode for now
```

Comment:
@HumairAK Do you think we will need this with v2 launch DSPA -- should we look to make it a field? Or is this something we want to get rid of and it's just a process now?

Reply:
The region thing is a bit tricky, we definitely need the user to specify the region, but we also need it in the `host` field  in the form of `s3.<region>.amazonaws.com`. This is all for aws s3.

(this is due to a bug I encountered in kfp, where it was resulting in some mismatched headers if we didn't specify the region in the endpoint for aws s3) 

I'm not very well versed on other s3 providers and how it works for them (for example gs/ceph/etc), it would be nice if we can figure out a list of providers that we support so I can go and test them all out, who has a region as required field or not.

Reply:
Ah this is interesting -- so we will need a story to chase this down -- Guillaume has been a good source, we should confer with him on the ticket.

@Gkrumbach07 mind logging a Task for this one too?

Reply:
https://issues.redhat.com/browse/RHOAIENG-2391